javascript/javascript1/week2/weatherwear.js

```diff
@@ -0,0 +1,17 @@
+//weatherwear
+
+function temperature(degrees) {
+  if (degrees <= 0) {
```

Comment:
if we have multiple if else with may condition, try to use switch condition in js.